app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,5 @@
+class DashboardController < ApplicationController
+  def index
+
```

Comment:
Thanks for the setup!

Reply:
No problem! Can't wait to see this built out